Rocket.Chat/Resources/cs.lproj/Localizable.strings

```diff
@@ -97,6 +97,7 @@
 // Socket Connection
 "connection.title" = "Sign in to your server"; // TODO
 "connection.button_connect" = "Connect"; // TODO
+"connection.server_url.placeholder" = "your-company.rocket.chat"; // TODO
```

Comment:
Should the placeholder look like a url at all? How about something like "Address for Rocket.Chat server" (or any other natural language string).

Reply:
@Sameesunkaria This was suggestion from @thiagosanchz... I think makes sense because you're telling the user how the entry should look like. 🤔